Libraries/Lists/VirtualizedSectionList.js

```diff
@@ -115,7 +91,8 @@ type OptionalProps<SectionT: SectionBase> = {
    */
   refreshing?: ?boolean,
 };
-
+/* $FlowFixMe - this Props seems to be missing a bunch of stuff. Remove this
```

Comment:
Can we fix this instead?

Reply:
I'm not really an expert with complex flow typing, I can try to remove this, but I'm not sure if I'm going to fix something of fix it in a good way :/